### ISSUE TITLE
Support Vitest

### ISSUE THREAD

**Issue by HelloAlexPan** [Pythagora-io/api#5]
Noticed that the prompts are all Jest related — is Vitest support planned?

**Comment by LeonOstrez** [comment]
Yeah, currently everything is for Jest. We will implement other frameworks very soon
